runtime-kernel/rpi-firmware-boot/autobuild/postinst

```diff
@@ -0,0 +1,58 @@
+#!/bin/bash
+
+if [ -d "/tree/runtime-kernel" ]; then
+  echo "Running in build environment, skipping firmware update."
+  exit 0
+fi
+
+if ! grep -q "/boot/rpi" /etc/fstab; then
+  echo "Mount point '/boot/rpi' does not exist in fstab file."
+  echo "You need to update firmware files manually, which is installed to /usr/lib/rpi64/boot."
+  echo "Or add the boot partition to fstab and try again."
+  exit 1
+fi
+
+if ! mount -a; then
+  # echo "Unable to mount /boot/rpi! Please check your fstab file. Not treating as an error, exiting cleanly."
+  echo "Failed to mount /boot/rpi! Please check your fstab file."
+  echo "Your firmware is not updated."
+  exit 1
+fi
+
+if [ -e /boot/rpi/kernel8.img ]; then
+  echo "Copying new firmware..."
+  cp -r /usr/lib/rpi64/boot/* /boot/rpi/
+elif [ -e /boot/rpi/kernel_2712.img ]; then
+  echo "Copying new firmware..."
+  cp -r /usr/lib/rpi64/boot/* /boot/rpi/
+else
+  echo "Copying kernel and firmware..."
+  cp -r /usr/lib/rpi64/boot/* /boot/rpi/
+  if [ -e /usr/lib/rpi64/kernel/kernel8.img ]; then
+    cp -r /usr/lib/rpi64/kernel/* /boot/rpi/
+  elif [ -e /usr/lib/rpi64/kernel/kernel_2712.img ]; then
+    cp -r /usr/lib/rpi64/kernel/* /boot/rpi/
+  fi
+fi
+
+if [ ! -e /boot/rpi/config.txt ]; then
+  echo "Creating config.txt..."
+  cp /usr/lib/rpi64/config/config.txt /boot/rpi/config.txt
+fi
+
+if [ ! -e /boot/rpi/distcfg.txt ]; then
+  echo "Creating distcfg.txt..."
+  cp /usr/lib/rpi64/config/distcfg.txt /boot/rpi/distcfg.txt
+fi
+
+if [ ! -e /boot/rpi/cmdline.txt ]; then
+  PARTUUID="$(lsblk -x MOUNTPOINT -o MOUNTPOINT,PARTUUID | egrep '^/\s' | awk '{print $2}')"
```

Comment:
```suggestion
  echo "Getting information of the root partition ..."
  eval "$(findmnt -Pyo PARTUUID /)"
```

Here's the reliable way to find the PARTUUID of the partition which the root filesystem resides on.